### ISSUE TITLE
Preview image for Vimeo already has a play button

### ISSUE THREAD

**Issue by Yokora** [distantnative/embed#84]
The preview image for my Vimeo videos already has a play button in it, so now I'm seeing two play buttons on all of my Vimeo videos. There's the transparent overlay, and then the one that's part of the image which is a slightly different shape. How would I go about removing the embed play button?

![image](https://user-images.githubusercontent.com/32977206/32702215-d66f5d2e-c7a0-11e7-942b-151a94713391.png)

**Comment by Michkkael** [comment]
I have the same issue.
I found this : https://vimeo.com/forums/help/topic:278001
But with the plugin, I don't know where it's the best place to put this "?background=1"

**Comment by Yokora** [comment]
I know you can type "?background=1" to the end of the url of the video you choose.
For example: https://vimeo.com/213041393?background=1
I tried this, and it didn't seem to work. I've read that this feature is supposed to only work for pro members of Vimeo and by default, it also makes your video auto-play when the page loads (which I don't want.)

It's strange because, using vimeo's embed code (instead of this embed plugin), the play button doesn't display in the middle of the video when it is large (but instead, the play button appears at the bottom left along with all of the other clutter.)

**Comment by Yokora** [comment]
For now, I'm using Youtube videos instead.

**Comment by distantnative** [comment]
Sorry guys, I have been very busy with working on Kirby 3. As I am doing this on the side of my regular job at a university, maintaining this plugin just doesn't work right now regrettably. I hope to have an update (bigger one also) as soon as Kirby 3 is out.

**Comment by Yokora** [comment]
No worries! I'm excited for the Kirby 3 update too!

**Comment by augenauf** [comment]
You can setup temporarily an transparently png in the config.
`c::set('plugin.embed.video.lazyload.btn', 'assets/plugins/embed/images/transparently.png');`

**Comment by tasinttttttt** [comment]
**tldr; Solution at the bottom.**
Its due to a function in the Embed library used by the plugin that selects the largest thumbnail image for a given embed.

The Embed library retrieves a 640px width thumb as the main thumbnail, then parses the OpenGraph meta tags. These tags contain several image urls, with being a 1280x720px image containing the play icon overlay.

The function naturally selects the second option. Which in this case is pretty bad.
But it's not a bug per se as Vimeo chose to use to put that weird play icon in their openGraph thumbnail...

**Solution**
Replace `plugins/embed/vendor/Embed/src/Adapters/Adapter.php` with this [Adapter.php](https://gist.github.com/tasinttttttt/21f69b6679a475b1263b39333beedf52)

A hack that works but I really didn't spend time optimizing or checking for weird exceptions.

In the file `plugins/embed/vendor/Embed/src/Adapters/Adapter.php`, within the `getImage()` function, 
I check if the first image url contains `vimeocdn`, if so I replace it with the url to the 1280 image, and then bypass the select the biggest image process.

Find it at line 447 of the file mentioned above.

Hope it helps
